src/components/Admin/TableEntry.vue

```diff
@@ -1,45 +1,58 @@
 <template>
   <q-spinner v-if="entryStore.isLoading" color="primary" size="2em" class="block q-mx-auto q-my-md" />
-  <q-table v-else :columns="columns" dense flat :filter="filter" hide-bottom hide-header :pagination="pagination" :rows="rows">
+  <q-table
+    v-else
+    flat
+    hide-bottom
+    :class="{ 'entries-table': !userStore.isEditorOrAbove }"
+    :columns="columns"
+    :dense="userStore.isEditorOrAbove"
+    :filter="filter"
+    :bordered="!userStore.isEditorOrAbove"
+    :hide-header="userStore.isEditorOrAbove"
+    :pagination="pagination"
+    :rows="rows"
+    :title="!userStore.isEditorOrAbove ? 'Manage Entries' : ''"
+  >
     <template v-slot:body-cell-actions="props">
       <td class="text-right">
         <q-btn
-          v-if="props.row.isWinner != true"
-          color="black"
-          :disable="userStore.getUser.role !== 'Admin'"
+          v-if="props.row.isWinner != true && userStore.isEditorOrAbove"
```

Comment:
Comparison props.row.isWinner != true may cause unexpected type coercion. Maybe replace it with "!=="